<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-07T22:28:33Z" name="Suite" finished-at="2020-02-07T22:29:40Z" duration-ms="67284">
    <groups>
    </groups>
    <test started-at="2020-02-07T22:28:33Z" name="Test" finished-at="2020-02-07T22:29:40Z" duration-ms="67284">
      <class name="com.apex.ecommerce.order.test.OrderTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-07T14:28:33Z" name="setUp" finished-at="2020-02-07T14:28:38Z" duration-ms="5278" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validatePidTest()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-07T14:28:38Z" name="validatePidTest" finished-at="2020-02-07T14:28:57Z" duration-ms="18929" status="FAIL">
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : Are you sure you want to remove this product from your cart}
  (Session info: chrome=80.0.3987.87): Are you sure you want to remove this product from your cart
Build info: version: '3.9.0', revision: '698b3178f0', time: '2018-02-05T14:26:55.441Z'
System info: host: 'Karthiks-MacBook-Air.local', ip: '2601:641:8000:e30:4096:dd0a:a4a5:29da%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '11.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 80.0.3987.87, chrome: {chromedriverVersion: 79.0.3945.36 (3582db32b3389..., userDataDir: /var/folders/gc/rt94tmz93zj...}, goog:chromeOptions: {debuggerAddress: localhost:64488}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: a7cc859ac01c19625468a0f95b28df46
*** Element info: {Using=xpath, value=//div[contains(text(),'Checkout')]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Are you sure you want to remove this product from your cart}
  (Session info: chrome=80.0.3987.87): Are you sure you want to remove this product from your cart
Build info: version: '3.9.0', revision: '698b3178f0', time: '2018-02-05T14:26:55.441Z'
System info: host: 'Karthiks-MacBook-Air.local', ip: '2601:641:8000:e30:4096:dd0a:a4a5:29da%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '11.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 80.0.3987.87, chrome: {chromedriverVersion: 79.0.3945.36 (3582db32b3389..., userDataDir: /var/folders/gc/rt94tmz93zj...}, goog:chromeOptions: {debuggerAddress: localhost:64488}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: a7cc859ac01c19625468a0f95b28df46
*** Element info: {Using=xpath, value=//div[contains(text(),'Checkout')]}
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy24.click(Unknown Source)
	at com.apex.ecommerce.order.page.OrderPage.productList(OrderPage.java:71)
	at com.apex.ecommerce.order.test.OrderTest.validatePidTest(OrderTest.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validatePidTest -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-07T14:28:58Z" name="close" finished-at="2020-02-07T14:28:59Z" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.order.test.OrderTest -->
      <class name="com.apex.ecommerce.registration.test.RegistrationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-07T14:29:28Z" name="setUp" finished-at="2020-02-07T14:29:33Z" duration-ms="4297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testCreateAccount()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-07T14:29:33Z" name="testCreateAccount" finished-at="2020-02-07T14:29:40Z" duration-ms="7538" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAccount -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-07T14:29:40Z" name="close" finished-at="2020-02-07T14:29:40Z" duration-ms="203" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.registration.test.RegistrationTest -->
      <class name="com.apex.ecommerce.signon.test.SignonTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:28:59Z" name="setUp" finished-at="2020-02-07T14:29:03Z" duration-ms="4810" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithAllBlanks()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:03Z" name="testWithAllBlanks" finished-at="2020-02-07T14:29:05Z" duration-ms="1135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithAllBlanks -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:05Z" name="close" finished-at="2020-02-07T14:29:05Z" duration-ms="184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:05Z" name="setUp" finished-at="2020-02-07T14:29:09Z" duration-ms="4414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithBlankUserId()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:09Z" name="testWithBlankUserId" finished-at="2020-02-07T14:29:10Z" duration-ms="1032" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithBlankUserId -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:10Z" name="close" finished-at="2020-02-07T14:29:10Z" duration-ms="205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:10Z" name="setUp" finished-at="2020-02-07T14:29:14Z" duration-ms="3237" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithInvalidCharacters()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:14Z" name="testWithInvalidCharacters" finished-at="2020-02-07T14:29:15Z" duration-ms="1308" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.apex.ecommerce.signon.test.SignonTest.testWithInvalidCharacters(SignonTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithInvalidCharacters -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:15Z" name="close" finished-at="2020-02-07T14:29:16Z" duration-ms="262" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:16Z" name="setUp" finished-at="2020-02-07T14:29:20Z" duration-ms="4009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithSuccessCredentials()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:20Z" name="testWithSuccessCredentials" finished-at="2020-02-07T14:29:21Z" duration-ms="1437" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithSuccessCredentials -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:21Z" name="close" finished-at="2020-02-07T14:29:21Z" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:21Z" name="setUp" finished-at="2020-02-07T14:29:26Z" duration-ms="4415" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithWrong()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:26Z" name="testWithWrong" finished-at="2020-02-07T14:29:27Z" duration-ms="1448" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [valid] but found [Invalid Username / Password!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [valid] but found [Invalid Username / Password!]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.apex.ecommerce.signon.test.SignonTest.testWithWrong(SignonTest.java:73)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithWrong -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-07T14:29:28Z" name="close" finished-at="2020-02-07T14:29:28Z" duration-ms="474" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.signon.test.SignonTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
